infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/logging/impl/Log.java

```diff
@@ -52,8 +52,8 @@
 	@Message(id = 1708, value = "Error during parse of Protobuf schema")
 	HibernateException errorAtProtobufParsing(@Cause Exception e);
 
-	@Message(id = 1709, value = "The remote cache '%s' was expected to exist but is not defined on the server")
-	HibernateException expectedCacheNotDefined(String cacheName);
+	@Message(id = 1709, value = "Error during caches start phase")
+	HibernateException errorAtCachesStart(@Cause Exception cause);
```

Comment:
When you introduce a new message error it should be a new method and code. We are not suppose to re-use existing codes. I'll update this